src/sqlfluff/dialects/dialect_snowflake.py

```diff
@@ -5258,7 +5238,9 @@ class GetStatementSegment(BaseSegment):
             Sequence(
                 "PATTERN",
                 Ref("EqualsSegment"),
-                Ref("QuotedLiteralSegment"),
+                OneOf(
+                    Ref("QuotedLiteralSegment"), Ref("ReferencedVariableNameSegment")
```

Comment:
Could we get a test case for `ReferencedVariableNameSegment` now this supports that?

Reply:
Good call!